crates/nu-command/src/network/http/client.rs

```diff
@@ -68,6 +81,19 @@ pub fn http_parse_url(
     Ok((requested_url, url))
 }
 
+pub fn http_parse_redirect_mode(mode: Option<Spanned<String>>) -> Result<RedirectMode, ShellError> {
```

Comment:
It's better to use `FromStr` trait: https://doc.rust-lang.org/std/str/trait.FromStr.html
```rust
impl FromStr for RedirectMode {
    ...
}
```
Then you can use `parse` method on `mode`

Reply:
Oh yup, that's so much smarter 😅 I blame not having written Rust in a while